rero_mef/agents/viaf/tasks.py

```
@@ -0,0 +1,179 @@
+        click.echo(msg)
+
+    if not online_data:
+        if verbose:
+            click.echo(f"  VIAF {pid}: not found online")
+        return Action.NOT_FOUND
+
+    # Handle redirect (VIAF cluster merge)
+    if redirect_from := online_data.get("redirect_from"):
+        existing_record = AgentViafRecord.get_record_by_pid(redirect_from)
+        if existing_record:
+            new_record, action, redirect_info = existing_record.handle_redirect(
```

Ruff (0.15.2)

[warning] Unpacked variable `new_record` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)